library/lib-core-logging/src/main/java/com/sokima/lib/corelogging/usecase/impl/ToStringMaskUseCaseImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.sokima.lib.corelogging.usecase.impl;
+
+import java.util.List;
+
+public class ToStringMaskUseCaseImpl extends MaskUseCaseImpl {
+    private static final String TO_STRING_REPLACEMENT_REGEX = "$1=" + MASK + "$3";
+    private static final String TO_STRING_PATTERN = "(%s)=([^\"]+?(, |\\)))";
```

Comment:
put this regex to ChatGpt, ask it to explain regex, and leave this as comment here

Regexes become unreadable even for its owners with time

Reply:
Done. Added also part for decimal and boolean.